forge-gui/res/cardsfolder/upcoming/rumbling_rockslide.txt

```diff
@@ -1,6 +0,0 @@
-Name:Rumbling Rockslide
```

Comment:
need to restore this card

Reply:
Yeah I asked him to remove it because it's a duplicate

Reply:
good catch :D